docs/layout-props.md

```diff
@@ -301,6 +301,16 @@ See https://developer.mozilla.org/en-US/docs/Web/CSS/bottom for more details of
 
 ---
 
+### `columnGap`
+
+`columnGap` property sets the size of the gap (gutter) between an element's columns. It works like `column-gap` in CSS. See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.
```

Comment:
One difference compared to what we link to on MDN is that we don't support percentages. We should maybe specify explicitly that the number here is pixels.